spikeextractors/extractors/yassextractors/yassextractors.py

```diff
@@ -71,19 +71,21 @@ def get_unit_spike_train(self, unit_id, start_frame=None, end_frame=None):
 
         if self.spike_train is None:
             self.spike_train = np.load(self.fname_spike_train)
-            
+        
         # find unit id spike times
-        idx = np.where(self.spike_train[:,1]==unit_id)
-        spike_times = self.spike_train[idx,0].squeeze()
+        idx = np.where(spike_train[:,1]==unit_id)[0]
```

Comment:
@catubc `spike_train` is not defined anywhere...you only define `self.spike_train`